src/main/scala/se/sics/concepts/examples/FigureOne.scala

```diff
@@ -0,0 +1,55 @@
+package se.sics.concepts.examples
+
+import org.apache.spark.{SparkContext, SparkConf}
+import se.sics.concepts.core.{CGraph, GraphBuilder, CKryoRegistrator}
+
+/**
+ * Running this program will create a json file as output, TODO(thvasilo): where?
+ * which you can visualize using D3.js TODO(thvasilo): Will we include vis tool?
```

Comment:
Output file in root, don't include vis tool (beyond scope of this repo), but possible link to website where vis is available.